### ISSUE TITLE
Example LTA signatures?

### ISSUE THREAD

**Issue by bseddon** [lovele0107/signature-conformance-checker#26]
Are you aware of any public source of existing signatures that match the LTA profile?  I am extending my XAdES to support LT and LTA signatures and  it will be great to be able to compare my results with a signature produced to these levels by software known to produce valid LTA signatures.  That is, include valid &lt;CertificateValues>, &lt;RevocationValues> and &lt;AchiveTimeStamp>.

So far I can produce an  &lt;AchiveTimeStamp> node that can be validated using the XAdESCC and the [EU Commission demo web app](https://ec.europa.eu/cefdigital/DSS/webapp-demo/validation).  I can see from sections 5.4.1 and 5.4.2 of the [ETSI EN 319 132-1 V1.1.1 (2016-04) specification](https://www.etsi.org/deliver/etsi_en/319100_319199/31913201/01.01.01_60/en_31913201v010101p.pdf) the content which **should** appear in the respective elements but ensuring it is the **correct** content, that's a different matter.

**Comment by bsanchezb** [comment]
Hello @bseddon ,

You can find some samples of XAdES signatures [within DSS project](https://github.com/esig/dss/tree/master/dss-xades/src/test/resources/validation). The directory contains not only LTA signatures, but also other levels (some of them valid, some of them not).
You can also find examples for signatures of other formats within the repository.

Best regards,
Aleksandr.

**Comment by bseddon** [comment]
Hi @bsanchezb I am currently participating in the 'plug test' which is running at the moment (until July 1st) and organized by ETSI and the authors of the XAdES standard.  Through the plug test now I have access to hundreds of examples and access to other signature validators that are able to validate the signatures created by the plugin.

**Comment by bsanchezb** [comment]
@bseddon , ok good to know!

P.S. we are also participants of the plugtest btw ;)

**Comment by bseddon** [comment]
@bsanchezb Yes, I see you at LU_NOW.  Thanks for the examples.  My contribution so far has been limited so it's great for me the end has been pushed out.
